.github/workflows/build-images-hotfixes.yaml

```diff
@@ -3,7 +3,7 @@ name: Hot Fix Image Release Build
 on:
   push:
     branches:
-      - hf/main/**
+      - hf/v1.15/**
```

Comment:
Maybe a question for @nbusseneau and @cilium/ci-structure , could we just add vX.Y matches into the main branch workflows for most of these, or do they need to exist on the stable branch?

Reply:
I think we can do it on `main`, however in this case we will only have a single version of the workflow and it must work for all branches.

If we don't expect much drift between build workflows it would be a good idea and would reduce backporting pressure as well.

Reply:
That'd be awesome. I had a variant on the same problem related to https://github.com/cilium/cilium/pull/27876 (& co.), because those workflows *only* exist on stable branches. Such workflows that are only on older releases complicate the stable branch creation workflow, making it more error prone.

Reply:
cc @pippolo84 for context on this question regarding PR 27876 and similar for older branches. It would be easier from a release management perspective to avoid workflows that are only on stable branches, or where the stable branch workflow significantly diverges from main. As a release manager, when I create a new stable branch, I would prefer not to have to go through and understand every different workflow that exists on main vs. the last stable branch in order to properly prepare the new stable branch.

Reply:
Filed https://github.com/cilium/cilium/pull/29902 to explore this specifically for the call-backport-label-updater workflow. Assuming we figure out the right details there, we can perhaps use that example as an example of how to update these other workflows.